engine/include/cubos/engine/assets/assets.hpp

```diff
@@ -356,6 +356,11 @@ namespace cubos::engine
         /// @return Lock guard.
         std::unique_lock<std::shared_mutex> lockWrite(const AnyAsset& handle) const;
 
+        /// @brief Checks if the given path is a valid path to an asset.
+        /// @param path Path to check.
+        /// @return A string that says if the given string is a path, uuid or not a valid format string
+        static std::string isPath(const std::string& path);
```

Comment:
Maybe this should be part of the `Asset` handle class, no?
But in any case, I would not use a `std::string` as a return type here: in these cases, either stick to booleans or create your own enums, otherwise it becomes very error prone.